run_witan_app.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# config
+cat <<EOF  > /root/.witan-app.edn
+{:cassandra-session {:host "cassandra-dcos-node.cassandra.dcos.mesos"
+                     :keyspace "witan"
+                     :replication 3}
+ :s3 {:bucket "witan-${ENVIRONMENT}-data"}}
```

Comment:
if you replace `${ENVIRONMENT}` with `${ENVIRONMENT?not defined}` you'll get a 'ENVIRONMENT not defined' message and failure to start if `${ENVIRONMENT}` evaluates to `""` - this prevents wierdness